<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TodayPage</class>
 <widget class="QWidget" name="TodayPage">
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>

   <!-- Плейсхолдер при отсутствии привычек -->
   <item alignment="Qt::AlignHCenter">
    <widget class="QWidget" name="noHabitsWidget">
     <layout class="QVBoxLayout" name="noHabitsLayout">
      <property name="alignment">
       <set>Qt::AlignHCenter</set>
      </property>
      <property name="spacing">
       <number>20</number>
      </property>

      <item>
       <widget class="QLabel" name="noHabitsLabel">
        <property name="text">
         <string>No habits for today or none created yet</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="noHabitsImage">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>

     </layout>
    </widget>
   </item>

   <!-- Скролл-область для карточек привычек -->
   <item>
    <widget class="QScrollArea" name="habitsScrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <layout class="QVBoxLayout" name="habitsLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>